neard/src/cli.rs

```diff
@@ -72,6 +72,15 @@ impl NeardCmd {
     }
 }
 
+fn unsafe_reset(command: &str, path: &std::path::Path, what: &str, default: &str) {
+    let dir =
+        path.to_str().map(|path| shell_escape::unix::escape(path.into())).unwrap_or(default.into());
+    warn!(target: "neard", "The ‘{}’ command is deprecated", command);
```

Comment:
Worth mentioning when it'll be removed. If somebody runs this command for the first time in Summer of 2022, they may not realize that this command will be come in a couple of months.